### DISCUSSION TITLE
I am not receiving emails notifications from issues tags

### DISCUSSION THREAD

**Discussion by GCH-JR** [community/community#45577] [Projects and Issues]
### Select Topic Area

General

### Body

I do not get any emails from my github account whether it is from an issue I have generated or being mentioned in another.
Does anybody have a solution to this problem?

**Comment by jge162** [comment]
@GCH-JR Please do the following.

1. Confirm that your email address is listed in your GitHub account settings.

2. Confirm that you have enabled notifications for the type of events you would like to receive emails for.

3. Check your spam folder for emails from GitHub.

4. If you are still not receiving emails, you may need to reach out to your email provider to confirm if they are blocking emails from GitHub.

5. Create a Webhook if you use discord, to get all notifications.
